src/System.Text.Primitives/System/Text/Parsers/InvariantBool.cs

```diff
@@ -12,7 +12,7 @@ public static partial class Parsers
     {
         public static partial class Utf8
         {
-            public unsafe static bool TryParseBoolean(byte* text, int length, out bool value)
+            unsafe static bool TryParseBoolean(byte* text, int length, out bool value)
```

Comment:
We should get rid of these APIs instead of hiding them (at least that was the conclusion from our API review - which didn't include the unsafe APIs that take pointers).
These were the APIs:
https://github.com/dotnet/apireviews/tree/master/2017/06-16-System.Text.Buffers
https://gist.github.com/ahsonkhan/6665d60cd41950dcbcb71f10a15886c9

Reply:
When we move the feature to corfx, we will just not move these APIs. I wanted to somehow keep them in case we need them in the future.

Reply:
That sounds good. In that case, should we mark it as internal, make internals visible to the test and keep the tests uncommented?